frontend/src/metabase/nav/containers/MainNavbar/DataAppNavbar/DataAppActionPanel/DataAppActionPanel.tsx

```diff
@@ -82,16 +87,21 @@ function DataAppActionPanel({
 
   return (
     <Root>
-      {hasEditPageAction && (
-        <Tooltip tooltip={t`Edit page`}>
-          <Button icon="pencil" onlyIcon onClick={onEditAppPage} />
-        </Tooltip>
-      )}
-      <EntityMenu
-        items={menuItems}
-        triggerIcon="ellipsis"
-        tooltip={t`Manage content, settings and more…`}
-      />
+      <ActionsContainer>
+        <NewButton onAddData={onAddData} onNewPage={onNewPage} />
```

Comment:
can we maybe get a tooltip on this puppy? when it's next to the other buttons, it might not be completely clear what you're adding.

![Screen Shot 2022-11-11 at 8 11 15 AM](https://user-images.githubusercontent.com/30528226/201368810-99380ad2-da59-4ad2-97db-850b5afbce6b.png)